homeassistant/components/lupusec.py

```diff
@@ -0,0 +1,97 @@
+"""
+This component provides basic support for Lupusec Home Security system.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/lupusec
+"""
+
+from datetime import timedelta
+import logging
+import voluptuous as vol
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers import discovery
+from homeassistant.const import (CONF_USERNAME, CONF_PASSWORD,
+                                 CONF_NAME, CONF_IP_ADDRESS)
+from homeassistant.helpers.entity import Entity
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['lupupy==0.0.10']
+
+DOMAIN = 'lupusec'
+
+NOTIFICATION_ID = 'lupusec_notification'
+NOTIFICATION_TITLE = 'Lupusec Security Setup'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Required(CONF_IP_ADDRESS): cv.string,
+        vol.Optional(CONF_NAME): cv.string
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+SCAN_INTERVAL = timedelta(seconds=2)
```

Comment:
This doesn't have an effect when just defining it in a component. In platforms it would have an effect and set the default scan interval for polling entities. Here it doesn't do anything since we're not actively using it.